contxt/utils/datetime.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
```

Comment:
i think this module might overshadow `datetime` imports for all other files in this directory. same thing about `collections`